.github/workflows/build.yaml

```diff
@@ -0,0 +1,53 @@
+name: build_and_test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: 3.8
+          channels: conda-forge,defaults
+          channel-priority: strict
+          show-channel-urls: true
+
+      - name: Install rubin-env
+        # rubin-env simplifies things so that eups, scons and a working
+        # compiler are installed.
+        shell: bash -l {0}
+        run: |
+          conda install mamba
+          mamba install -y -q rubin-env
```

Comment:
It takes 2.5 minutes and 650MB to install rubin-env. Maybe I should ask to just install scons, eups, and a compiler. What would the compiler be called?

Reply:
You can do `mamba install compilers` and you'll get `c-compiler`, `cxx-compiler`, and `fortran-compiler`.  Or just choose a subset.

Reply:
Thanks. Now 48 seconds.